### DISCUSSION TITLE
How do I specify output format via eslintrc?

### DISCUSSION THREAD

**Discussion by Hultner** [eslint/eslint#15118] [Q&A] [answered]
I need to specify output format as JSON via my .eslintrc file but I can't seem to figure out what to put in the file to achieve this. I've tried searching the documentation, GitHub and Google but I can't seem to find any relevant information.

How do I specify `--format json` via `.eslintrc`?

Any help would be highly appreciated.

**Comment by nzakas** [comment, marked as answer]
This isn’t possible. You must specific output format on the command line.
